packages/project-build/src/db/folders.ts

```diff
@@ -0,0 +1,11 @@
+import type { Folder, Folders } from "@webstudio-is/sdk";
+
+export const parseFolders = (foldersString: string): Folders => {
```

Comment:
See build.ts. i started using universal converters to avoid this boilerplate

Reply:
ok then it should have been clear from the code base - deprecation notices are required for us to understand whats new whats old